modules/host/negotiateformcontract.go

```diff
@@ -128,6 +129,7 @@ func (h *Host) managedRPCFormContract(conn net.Conn) error {
 	// Send the host settings to the renter.
 	err := h.managedRPCSettings(conn)
 	if err != nil {
+		h.log.Debugln("RPCSettings call failed during form contract:", err)
 		return err
```

Comment:
Is the question general to all the logging statements, or just this one?

By making a logging statement, we get the line of code that is declaring the logging. A lot of the messages across the files are very similar, so having the exact file location, while fragile to changes in the codebase, give us a more complete idea of what is happening, especially if the message itself ends up being not nearly specific enough